importers/scripts/importOsisTest.ts

```diff
@@ -0,0 +1,53 @@
+// import { V11nImporter } from './../src/stepdata/v11n-rules/index';
+// import { SwordImporter } from './../src/bible/sword/src/importer';
```

Comment:
remove commented out code from this file

Reply:
since this is a file for manual dev testing i think it can be justifiable to leave some commented code for easy switching

Reply:
removed comments, added configuration parameters to `run` method